### ISSUE TITLE
Make the roles.json and nodes.json files public

### ISSUE THREAD

**Issue by gravitystorm** [osmfoundation/osmf-server-info#9]
The readme says "Currently the _data/nodes.json and _data/roles.json are not publicly available."

This makes it impossible for anyone to help develop this codebase. When I was working with those files, I didn't see anything that I thought was sensitive information, but I might easily have missed something. Any thoughts @tomhughes?

**Comment by tomhughes** [comment]
Well roles shouldn't be an issue, because it's all in the public repo anyway.

Nodes is pretty much impossible to verify I fear, and to be sure that something random won't change on a chef upgrade (or more precisely an ohai upgrade) that could wind up leaking something sensitive.

**Comment by abitrolly** [comment]
Can there be an example of these files at least?

**Comment by tomhughes** [comment]
Not easily - the node data is a huge amount of automatically generated data for each machine combined with specific customisation from the cookbooks. You can't really write a dummy node.

**Comment by abitrolly** [comment]
https://github.com/gravitystorm/osmf-server-info/blame/master/index.md uses only a subset of data. Is it possible to filter this `node.json` to export only these?

- `node.automatic.roles`
- `node.name`
- `node.automatic.ohai_time`
- `node.default.hosted_by`
- `node.default.location`
- `node.override.country`

Looks like there is no other info being used.

**Comment by tomhughes** [comment]
That's just not true - that might be all the index uses but other pages use a lot more.

What exactly are you trying to do? I'm not really sure why anybody that is an OSM system administrator would want to make a change here? Pretty much anything you do here needs fairly deep knowledge of our infrastructure and often a lot of experimentation to find (or make available) the necessary details.

**Comment by abitrolly** [comment]
@tomhughes need the same feature for the local community - https://github.com/osm-by/OpenStreetMap.by/pull/17

**Comment by tomhughes** [comment]
So are you using chef to manage your servers? or at least gathering data from them with ohai? Because if you're not then I doubt this repository is going to help...

**Comment by abitrolly** [comment]
@tomhughes I don't think that Chef is a suitable model for the servers owned by different people who may want to run their own payload there as well. I think about them gathering data and sending them to some endpoint, with `ohai` or Ansible facts, or some kind of JSON version of `neofetch`.
